result

```diff
@@ -0,0 +1 @@
+/nix/store/nw0mm1z2afpxv5rgf6w3apxc8pxwgd6h-idris2-0.3.0
```

Comment:
You probably would not want to check the `result` symlink into the repository, no?

Reply:
This was removed in a later commit (#953)